digitalocean/database/resource_database_mongodb_config.go

```diff
@@ -0,0 +1,174 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/digitalocean/godo"
+	"github.com/digitalocean/terraform-provider-digitalocean/digitalocean/config"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceDigitalOceanDatabaseMongoDBConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDigitalOceanDatabaseMongoDBConfigCreate,
+		ReadContext:   resourceDigitalOceanDatabaseMongoDBConfigRead,
+		UpdateContext: resourceDigitalOceanDatabaseMongoDBConfigUpdate,
+		DeleteContext: resourceDigitalOceanDatabaseMongoDBConfigDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDigitalOceanDatabaseMongoDBConfigImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"connect_default_read_concern": {
```

Comment:
Looks like a typo here:

```suggestion
			"default_read_concern": {
```